.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Do we push .vscode folder to git? If not, why so and what all files we should not push?